src/_components/RoutesComponent.jsx

```diff
@@ -2,11 +2,11 @@ import React, { Component } from 'react';
 import { Route, Switch } from 'react-router-dom';
 
 import AuthenticateComponent from './AuthenticateComponent';
-import AssetTypes from '../components/AssetTypesComponent';
+import AssetTypesComponent from '../components/AssetTypesComponent';
```

Comment:
I believe changes to this file are unnecessary. Please revert to what is in develop